src/config.ts

```diff
@@ -0,0 +1,7 @@
+import * as vscode from "vscode";
+
+/** Get the value of a workspace config property */
+export const getConfig = <T>(extensionId: string, config: string): T => {
+  const value = vscode.workspace.getConfiguration(extensionId).get<T>(config);
+  return value as T;
```

Comment:
Two things:

1. is it really ok to just assume that its not `undefined`?
2. why not return it directly and use the `as T` at the end (e.g. `return vscode.workspace.getConfiguration(extensionId).get<T>(config) as T;`)

Reply:
The component which calls this function should handle it when it's undefined. I adjusted the return type here.